### PR TITLE
Handle stories with missing contentRating, storyStatus & support beta fictionlive.com domain

### PR DESCRIPTION
**Pull request by HazelSh**

### CODE REVIEW

**Comment by HazelSh** [comment]
should close #547

File: fanficfare/adapters/adapter_fictionlive.py [hunk 1]
```diff
@@ -116,6 +114,21 @@ def extract_metadata(self, data, get_cover):
 
         # nearly everything optional from here out
 
+        if 'storyStatus' in data:
+            self.story.setMetadata('status', data['storyStatus'])
+        elif 'complete' in data:
+            if data['complete'] == True:
+                self.story.setMetadata('status', "finished")
```
Comment (JimmXinu) [inline fanficfare/adapters/adapter_fictionlive.py:121]: Please use `Completed` and `In-Progress` for `status`.

While `status` can be set to other values as well, FFF assumes that `Completed` and `In-Progress` are standard and there's code in the plugin for setting boolean columns that expects them.

Comment (HazelSh) [reply fanficfare/adapters/adapter_fictionlive.py:121]: Ok, will do. So should I always translate what the site gives, then? If so, what's the equivilent of `hiatus`?

Comment (JimmXinu) [reply fanficfare/adapters/adapter_fictionlive.py:121]: `Completed` and `In-Progress` are the only 'standard' values.  Personally, I'd capitalize any other values (Python's `title()` should be sufficient), but I won't insist on it.

**Comment by JimmXinu** [comment]
FYI, #552 also touched `adapter_fictionlive.py`.  You probably should rebase or merge that change.

**Comment by HazelSh** [comment]
please ignore all the git nonsense, I've been having some issues. clean pull request incoming.

**Pull request closed without merging**